src/api/get.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+module.exports = (send) => {
+  return function get (path, archive, compress, compressionLevel, cb) {
+    if (archive === true && typeof compress === 'function') {
```

Comment:
Or since `archive`, `compress`, `compressionLevel` are all optional, why not pass them on the `opts` object?

I think that you can just return the `argCommand` and it will [take care of that](https://github.com/ipfs/js-ipfs-api/blob/master/src/cmd-helpers.js#L14) :)